docs/guides/dkms.md

```diff
@@ -134,6 +134,36 @@ It unloads the Touchbar modules as they can cause issues for suspend.
 
 Your keyboard backlight may remain switched off on resuming and backlight controls may stop working. A restart fixes the backlight controls. You may also run `echo 60 > /sys/class/leds/apple::kbd_backlight/brightness` to turn on the backlight to the maximum level if you do not want to boot. Replace 60 with a lower number for lower brightness.
 
+You can also control the backlight manually by binding the backlight controls directly to the kbd_backlight by using `acpilight`.
+
+This package is available on Arch Linux by installing `sudo pacman -S acpilight`.
+On other distributions, you are required to run `git clone https://gitlab.com/wavexx/acpilight && cd acpilight && sudo make install` Make sure that you have the packages `git` and `make` installed.
```

Comment:
Maybe we could just tell them to follow the instructions on the readme of the acpilight repo? Also there may be other distros that package acpilight so I don't think we can say for sure that for all other distros users are required to install if from source.

I'm also not sure if we should be making users install acpilight, similar programs [light](https://github.com/haikarainen/light) and [brightnessctl](https://github.com/Hummer12007/brightnessctl) have been updated more recently and also have instructions on how to install them on different distros including getting udev rules working so they don't need as much documentation on this wiki for users to use them.